src/__tests__/prompt/BasePrompt.spec.ts

```diff
@@ -73,4 +74,39 @@ describe('BasePrompt', () => {
       expect(cursor).toBe(richCursor)
     })
   })
+  
+  describe('block', () => {
+    it('should return block when block exists on runner', () => {
+      const config: IPromptConfig<any> & IBasePromptConfig = dataset._prompts[0]
+      const
+        ctx = dataset.contexts[1] as IContextInputRequired,
+        runner = new FlowRunner(ctx),
+        prompt = new MessagePrompt(
+          config as IMessagePromptConfig & IBasePromptConfig,
+          '09894745-38ba-456f-aab4-720b7d09d5b3', // first interaction
+          runner)
+
+      expect(prompt.block).toBe(ctx.flows[1].blocks[0])
+    })
+    
+    it.skip('should return block on flow specified by interaction and not necessarily active flow', () => {
```

Comment:
This is a feature of this right? Should the test be made for it?

Reply:
Yes! I skipped because I was finding it hard to set up for it. 
Can I add this after @bulatgab and I have had a chance to integrate with his work?

Reply:
For sure!